txbits/public/js/password-form.js

```diff
@@ -41,5 +41,21 @@ $(function(){
         }
         $password1.keyup(check_form);
         $password2.keyup(check_form);
+    }).submit(function(e) {
+        var $this = $(this);
+        var min_len = $this.attr('data-password-length');
```

Comment:
I did it in 4d4007fb7cf490446ee0f6f383be69f90f4bed25. I found a lot of inconsistancies while I was doing it. I think it's okay now.